### PR TITLE
added a function which returns a hash of the required fields

### PR DESCRIPTION
**Pull request by Asergey91**
So i made the hashing function however unsure what predicate to use

### CODE REVIEW

File: support/pre-importer.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // @ts-ignore
-import {update, sparqlEscapeUri, sparqlEscapeString, sparqlEscapeDateTime, uuid} from  'mu';
+import {update, sparqlEscapeUri, sparqlEscapeString, sparqlEscapeDateTime, uuid, query} from  'mu';
 import {prefixMap} from "./prefixes";
+import {createHash} from 'crypto';
```
Comment (nvdk) [inline support/pre-importer.js:4]: is this a library? I guess it needs to be added to the package.json?

File: support/pre-importer.js [hunk 2]
```diff
@@ -15,12 +16,65 @@ function hackedSparqlEscapeString( string ) {
...
+    
+    return hashString;
+  }
+  catch(error){
+    throw new Error("unable to sign resource because couldn't find relavant data in the database");
+  }
+}
+
 async function handleVersionedResource( type, versionedUri, sessionId, targetStatus, customSignaturePredicate, customStatePredicate, customContentPredicate ) {
+  const now=new Date();
```
Comment (nvdk) [inline support/pre-importer.js:71]: place mind the spacing ;)

File: support/pre-importer.js (hunk 1 above)
Comment (Asergey91) [reply support/pre-importer.js:4]: its part of node.js

Comment (Asergey91) [reply support/pre-importer.js:4]: https://nodejs.org/api/crypto.html#crypto_crypto

Comment (Asergey91) [reply support/pre-importer.js:4]: not sure if this might be a problem on an external system

**Review by nvdk — APPROVED** [review]
I've added some basic documentation in the readme, and changed sign:hashCode to sign:hashValue as that was already used in mu-cl-resources. but other than that looks fine to me

**Pull request merged**